modules/View.js

```diff
@@ -191,20 +195,25 @@
         return $.Deferred().resolve().promise();
       }
       this.__updateInjectionSiteMap();
-      this.trigger('render:before-dom-update');
+      this.trigger('render:before-detach-views');
       this.detachTrackedViews();
+      this.trigger('render:before-dom-update');
```

Comment:
this moved, are there implications we need to worry about?

Reply:
reverted